detox/src/android/actions/native.js

```diff
@@ -81,10 +81,10 @@ class ScrollAmountStopAtEdgeAction extends Action {
 }
 
 class ScrollEdgeAction extends Action {
-  constructor(edge) {
+  constructor(edge, startPositionX = -1, startPositionY = -1) {
```

Comment:
This is somewhat confusing for me. Documentation says that `NaN` acts like `auto`, but if someone indeed decides to pass that `NaN` explicitly, this implementation will not hold water.

I'm not sure if this should be addressed in this PR, but overall this is a bug.

Reply:
Confusing indeed, but how is that a bug @noomorph?
If someone will pass NaN, `startPositionX` will be converted to `null` in JSON, and Java will handle it the same as it will handle the default `-1` value passed from here, see: https://github.com/wix/Detox/blob/4332464d6b3e8fdb0188cfe04a35100ede94ac6c/detox/android/detox/src/full/java/com/wix/detox/espresso/DetoxAction.java#L88
please correct me if I'm wrong @d4vidi


I generally agree with @noomorph , it's a bit confusing that we use both `-1` or `NaN` for the same behaviour. However it's a copied code from different methods we have that does exactly the same, we can probably replace the default `-1` with `NaN` and avoid the `< 0` check from the line I mentioned.

Reply:
I was left under the impression that passing NaN directly won't work – otherwise, why would Android implement such a weird check for `< 0`?

In any case, I stated that this issue can be addressed in a separate PR.